src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AsylumCaseStatusSearchHandlerTest.java

```diff
@@ -28,12 +29,12 @@
     @Mock private Callback<AsylumCase> callback;
     @Mock private CaseDetails<AsylumCase> caseDetails;
     @Mock private AsylumCase asylumCase;
-
+    @Mock private HomeOfficeCaseStatusClient homeOfficeCaseStatusClient;
```

Comment:
You're adding a mock but not using it anywhere?  Would indicate that your test needs updating maybe?

Reply:
It is required for creating the AsylumCaseStatusSearchHandler